source/sass/abstracts/_variables.scss

```diff
@@ -15,26 +15,71 @@ $font-family-primary: 'Muli', sans-serif;
 $font-logo: 'Baumans', cursive;
 
 
+
 //Colors
 ////////////////////////////////////////////////////////////////
 
+$white:  #fff;
+$gray-100: #f8f9fa;
+$gray-200: #e9ecef;
+$gray-300: #dee2e6;
+$gray-400: #ced4da;
+$gray-500: #adb5bd;
+$gray-600: #868e96;
+$gray-700: #495057;
+$gray-800: #343a40;
+$gray-900: #212529;
```

Comment:
using Sass comman to create gray

Reply:
Temporary. Will research sass command for creating variables in future